src/TLuaInterpreter.cpp

```diff
@@ -2462,9 +2472,15 @@ int TLuaInterpreter::openUserWindow( lua_State *L )
     {
         luaSendText = lua_tostring( L, 1 );
     }
+
+    bool loadLayout = true;
+    if( ! lua_isnoneornil(L, 2) && lua_isboolean(L, 2) ) {
+        loadLayout = lua_toboolean(L, 2);
+    }
+
     Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
     QString text(luaSendText.c_str());
-    mudlet::self()->openWindow( pHost, text );
+    mudlet::self()->openWindow( pHost, text, loadLayout );
```

Comment:
I think there may be scope to pass back the boolean result that I think that `mudlet::openWindow(...)` has...